### PR TITLE
Add more debug logs to addLogicalPort

### PR DESCRIPTION
**Pull request by tssurya**
Signed-off-by: Surya Seetharaman <suryaseetharaman.9@gmail.com>

**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: go-controller/pkg/ovn/pods.go [hunk 1]
```diff
@@ -313,10 +313,13 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) (err error) {
...
 	}
 
+	var libovsdbExecuteTime time.Duration
+	var podAnnoTime time.Duration
 	// Keep track of how long syncs take.
 	start := time.Now()
 	defer func() {
-		klog.Infof("[%s/%s] addLogicalPort took %v", pod.Namespace, pod.Name, time.Since(start))
+		klog.Infof("[%s/%s] addLogicalPort took %v, libovsdbExecuteTime %v, pod Annotation time: %v",
+			pod.Namespace, pod.Name, time.Since(start), libovsdbExecuteTime, podAnnoTime)
```
Comment (tssurya) [inline go-controller/pkg/ovn/pods.go:322]: @dcbw : am I missing something? I remember you had added more verbose logs in one of your scale image test runs...wondering if there is something out of that worth capturing here by default?

Comment (dcbw) [reply go-controller/pkg/ovn/pods.go:322]: I think these are the two key ones. Everything else is tiny compared to these two.

But how about:

libovsdbExecuteTime -> libovsdb time
pod Annotation time -> annotation time

?

Comment (tssurya) [reply go-controller/pkg/ovn/pods.go:322]: ack let me fix the variable names, I like your suggestions better than the current ones.

Comment (tssurya) [reply go-controller/pkg/ovn/pods.go:322]: oh I think you meant the logs not variables, anyways will change both :D

**Pull request merged**